lib/polyfill/patchedmediakeys_webkit.js

```diff
@@ -32,24 +32,20 @@ shaka.polyfill.PatchedMediaKeysWebkit = class {
    * @export
    */
   static install() {
+    if (!shaka.util.Platform.willMediaKeysWebkitBePolyfilled()) {
```

Comment:
I don't love moving the details of the detection out of the polyfill.  If there's a need to detect whether or not a polyfill was installed, perhaps we can compare the implementation to native instead.  This appears to work:

```js
const polyfilled =
    navigator.requestMediaKeySystemAccess !=
    Navigator.prototype.requestMediaKeySystemAccess;
```

If you need to identify a specific polyfill, you could perhaps have a tag or something on the implementation without needing to depend on the polyfill from some part of the core code.

Reply:
Well it doesn't really work because there is also the EMEEncryptionScheme polyfill that overrides the requestMediaKeySystemAccess. I'll add a tag on the polyfill implementation so we can detect shaka own implementation